libs/wingsdk/src/target-tf-aws/function.ts

```diff
@@ -227,6 +227,7 @@ export class Function extends cloud.Function implements IAwsFunction {
         : Duration.fromMinutes(1).seconds,
       memorySize: props.memory ?? DEFAULT_MEMORY_SIZE,
       architectures: ["arm64"],
+      reservedConcurrentExecutions: props.concurrency ?? DEFAULT_CONCURRENCY,
```

Comment:
I don't think we need to set the default to 10 because there's a major cost implication here.

```suggestion
      reservedConcurrentExecutions: props.concurrency,
```